packages/synapse-react-client/src/components/MissingQueryResultsWarning/MissingQueryResultsWarning.tsx

```diff
@@ -43,15 +43,18 @@ const MissingQueryResultsWarning: React.FunctionComponent<
     partMask: SynapseConstants.BUNDLE_MASK_QUERY_COUNT,
   }
 
-  const { data: queryResult } = useGetQueryResultBundle(request, {
-    enabled: isMissingResultsCalculable,
-  })
+  const { data: queryResult } = useGetQueryResultBundleWithAsyncStatus(
```

Comment:
`useGetQueryResultBundle` is deprecated in favor of `useGetQueryResultBundleWithAsyncStatus`.